### PR TITLE
Correct headers of scripts for release

### PR DESCRIPTION
**Pull request by SabineHaas**
In this PR we check the headers of the scripts: correct inputs, output and description.

In detail, we check that:
- Inputs are consistent with the sys.argv in the script
- Outputs are consistent with the sys.argv in the script
- Paths are consistent with Snakefile
- Description is correct and helpful
- docstring is rendered in docs

For these scripts
- [x] build_datapackage.py @jnnr 
- [x] create_empty_scalars.py @jnnr
- [x] create_input_data_overview.py @jnnr
- [x] join_scenarios.py @jnnr
- [x] optimize.py @jnnr
- [x] plot_conv_pp_scalars.py @MaGering
- [x] plot_dispatch.py @monika-o 
- [x] plot_joined_scalars.py @monika-o 
- [x] postprocess.py @jnnr
- [x] prepare_conv_pp.py @SabineHaas 
- [x] prepare_electricity_demand.py @MaGering 
- [x] prepare_feedin.py @SabineHaas 
- [x] prepare_heat_demand.py @MaGering 
- [x] prepare_re_potential.py @SabineHaas 
- [x] prepare_scalars.py @SabineHaas 
- [x] process_re_potential.py @SabineHaas 
- [x] tools/dataprocessing @MaGering 
- [x] prepare_vehicle_charging_demand.py @SabineHaas

### CODE REVIEW

**Comment by SabineHaas** [comment]
[`prepare_scalars`](https://oemof-b3.readthedocs.io/en/features-update-docs/model_pipeline.html#preprocessing) is not shown in model pipeline and in [`prepare_cov_pp`](https://oemof-b3.readthedocs.io/en/features-update-docs/preprocessing/prepare_conv_pp.html) there is no entry.

File: scripts/process_re_potential.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 input_dir : str
     ``results/_resources/RE_potential/``: Path to input directory of renewable potential
```
[the remark this answers is not in the document]
Comment (SabineHaas) [reply scripts/process_re_potential.py:6]: No, it's the only input. The second input is the script.

File: scripts/plot_conv_pp_scalars.py [hunk 2]
```diff
@@ -2,7 +2,10 @@
 Inputs
 -------
 resources : str
-    path of input file with data of prepared conventional power plant data as .csv
+    ``results/_resources/{resource}.csv``: path of input file with data of prepared conventional
+    power plant data as .csv
+var_name :
+    ``scripts/plot_conv_pp_scalars.py``: The present script
```
[the remark this answers is not in the document]
Comment (MaGering) [reply scripts/plot_conv_pp_scalars.py]: Solved with commit https://github.com/rl-institut/oemof-B3/pull/158/commits/493cf05872e075c292b6ddb013079a162a19f4d8.

**Comment by jnnr** [comment]
> [`prepare_scalars`](https://oemof-b3.readthedocs.io/en/features-update-docs/model_pipeline.html#preprocessing) is not shown in model pipeline and in [`prepare_cov_pp`](https://oemof-b3.readthedocs.io/en/features-update-docs/preprocessing/prepare_conv_pp.html) there is no entry.

On my local build, both are shown. RTD builds are broken and will be fixed in #160.

**Comment by jnnr** [comment]
Thank you @MaGering, @SabineHaas, @monika-o!

**Pull request merged**